src/p_mobj.cpp

```diff
@@ -8245,6 +8245,45 @@ DEFINE_ACTION_FUNCTION(AActor, Vec2Angle)
 	ACTION_RETURN_VEC2(self->Vec2Angle(length, angle, absolute));
 }
 
+DEFINE_ACTION_FUNCTION(AActor, Vec2Diff)
+{
+	PARAM_SELF_PROLOGUE(AActor);
```

Comment:
Static function needs `PARAM_PROLOGUE()` macro. Same for `Vec3Diff()` function.
Please use Debug configuration for developing because this line will trigger an assertion failure.